views/lexicon/tabs/progress_bars.html.twig

```diff
@@ -216,5 +216,32 @@
         </a>
 
     </div>
+    <br/>
+    <h2 class="heading">Stepped Progress Bar</h2>
+
+    <div class="stepped-progressbar">
+        <div class="step"><span class="step-box current">1</span></div>
```

Comment:
Any reason why we aren't using an `ol` here? Seem more semantically correct.

Reply:
Don't think it makes any real difference based on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol

Reply:
Visually there wouldn't be, but to a screen reader would announce it as a list and depending on the screen reader announce it as a progress bar (if we used appropriate aria attributes and `aria-valuetext` etc).